src/templates/app/user/metatrader.jsx

```diff
@@ -308,33 +317,52 @@ const Metatrader = () => (
+                                    <p id='msg_form' className='invisible error-msg' />
+                                    <FormRow
+                                        is_two_rows
+                                        type='password'
+                                        id='txt_main_pass'
+                                        label={it.L('Password')}
                                     />
+                                    <div className='gr-padding-10 center-text'>
+                                        <ResetPasswordButton />
+                                        <button
+                                            id='btn_submit_new_account'
+                                            type='submit'
+                                            action='new_account'
+                                            className='button button-secondary'
+                                        >{it.L('Next')}
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                            <div id='view_3' className='gr-row invisible'>
+                                <div className='container gr-8 gr-12-m center-text'>
+                                    <p id='password_reset_error' className='invisible error-msg center-text' />
+                                    <ResetPasswordButton />
+                                </div>
+                            </div>
+                            <div id='view_4' className='gr-row invisible'>
+                                <div className='container gr-8 gr-12-m'>
+                                    <h2>{it.L('Reset password')}</h2>
+                                    <p className='notice-msg center-text'>{it.L('Please check your email for further instructions.')}</p>
                                 </div>
                             </div>
                         </div>
                     </form>
                 </div>
                 <div id='frm_manage_password'>
                     <div className='gr-row'>
-                        <div className='gr-6 gr-12-m flex'>
+                        <div className='gr-12 gr-12-m flex'>
                             <div className='mt-panel mt-container'>
                                 <form id='frm_password_change'>
                                     <div className='center-text hint gr-padding-20 gr-parent'>
                                         <h3 className='secondary-color'>{it.L('Change password')}</h3>
                                     </div>
-
+                                    <p className='notice-msg'>
+                                        {it.L('We’ve upgraded our system to support a single, more secure password across all of Deriv/Binary.com. Once you’ve set a new password, you can use it to log into all your Deriv/Binary.com, and DMT5/MT5 accounts.')}
```

Comment:
please use it.website_name for Binary.com

Reply:
This is also removed, should we do the same for texts like `Deriv/Binary.com`? (A place holder of some sort?)